payment_stripe_klarna/models/payment_stripe.py

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+
+from collections import namedtuple
+import logging
+from odoo import api, fields, models, _
+
+_logger = logging.getLogger(__name__)
+
+# The following currencies are integer only, see https://stripe.com/docs/currencies#zero-decimal
+INT_CURRENCIES = [
+    u'BIF', u'XAF', u'XPF', u'CLP', u'KMF', u'DJF', u'GNF', u'JPY', u'MGA', u'PYG', u'RWF', u'KRW',
+    u'VUV', u'VND', u'XOF'
+]
+STRIPE_SIGNATURE_AGE_TOLERANCE = 600  # in seconds
+
+
+class PaymentAcquirerStripe(models.Model):
+    _inherit = 'payment.acquirer'
+
+    @api.model
+    def _add_available_payment_method_types(self, stripe_session_data, tx_values):
```

Comment:
Have you any attempt been made to call super(...) ? stripe_session_data can be extended nicely as it is a dict.
yes: I acknowledge the new one will basically be the old but for Klarna only but without super() we break any other extensions and changes.